config/routes.rb

```diff
@@ -4,6 +4,6 @@
   resources :movies, only: %i[index] do
     get :recommendations, on: :collection
     get :user_rented_movies, on: :collection
-    get :rent, on: :member
+    post :rent, on: :member
```

Comment:
Since this method actually creates a rent object it should be a `POST` instead of a `GET` HTTP method.